server/app.js

```diff
@@ -2,6 +2,8 @@ const express = require("express");
 const morgan = require("morgan");
 const cors = require("cors");
 const bodyParser = require("body-parser");
+const fs = require('fs');
+const { Question } = require("./api/models/question");
```

Comment:
aren't we just exporting the Question model from the file? Do we really need `{}`

Reply:
Yes we need it. Please see this https://github.com/UdacityMobileWebScholarship/guess-quote/blob/49181c9fc821e9e8a69c974911c5919d9b0c9e9d/server/api/models/question.js#L38

Reply:
Ok then let's export it directly. Without the braces.